posthog/queries/sessions.py

```diff
@@ -167,7 +169,7 @@ def _determineInterval(interval):
                 "aggregated_value": int(avg_split[0]),
             }
         )
-        time_series_data.update({"chartLabel": "Average Duration of Session (seconds)"})
+        time_series_data.update({"chartLabel": "Average Duration of Session ({})".format(label)})
```

Comment:
I think we could standardize to "Average Session Length" here while we're at it, since that's what the Type selector says.

Reply:
sounds good made the change